units/generate_tasks.go

```diff
@@ -113,93 +130,43 @@ func (j *generateTasksJob) generate(ctx context.Context, t *task.Task) error {
 	}
 
 	var projects []model.GeneratedProject
-	projects, err = parseProjectsAsString(t.GeneratedJSONAsString)
+	projects, err = parseProjectsAsString(ctx, t.GeneratedJSONAsString)
 	if err != nil {
 		return errors.Wrap(err, "parsing JSON from `generate.tasks`")
 	}
-	grip.Debug(message.Fields{
-		"message":       "generate.tasks timing",
-		"function":      "generate",
-		"operation":     "parseProjects",
-		"duration_secs": time.Since(start).Seconds(),
-		"task":          t.Id,
-		"job":           j.ID(),
-		"version":       t.Version,
-	})
-	start = time.Now()
```

Comment:
do we want to keep both for a bit or are we confident enough that the honeycomb trace will be a sufficient replacement?

Reply:
I think it should be fine to go straight to honeycomb